distribution/src/main/resources/bin/elasticsearch

```diff
@@ -56,6 +56,15 @@ fi
 CDPATH=""
 SCRIPT="$0"
 
+COMMAND="$1"
+if [[ -z "$COMMAND" ]]; then
+    COMMAND="start"
+elif [ "$COMMAND" = "start" ] || [ "$COMMAND" = "version" ]; then
```

Comment:
@dakrone By the way, my preference would be to not be lenient here (i.e., assume `start` if no command is provided but I doubt that I will be able to convince others of that).